egs/wsj/s5/utils/data/combine_short_segments.sh

```diff
@@ -158,7 +158,7 @@ while changed:
                  changed = True
 
 for uniq in sorted(uniq2orig_uniq.keys()):
-    print uniq, uniq2orig_uniq[uniq]
+    print(uniq, uniq2orig_uniq[uniq])
```

Comment:
Can you please also change the python invocation to python3?  Otherwise this will break systems with python2 as the default.

Reply:
I used python3 and got this error. Kaldi require python2 as the default?
Or please tell me the script instead for this.

Reply:
I was talking about line 137, change python to python3

Reply:
I explicitly changed python to python3 and got the same error.
Should I use python2?

Reply:
I ran this by python2.
In this case, it might be due to using conda python. I try to inspect this error.
If I clear this error,  I do another PR.
thx.